test/se_manager/.cache/selenium/se-metadata.json

```diff
@@ -1,15 +1,31 @@
 {
-  "browsers": [],
+  "browsers": [
+    {
+      "browser_name": "chrome",
+      "major_browser_version": "132",
+      "browser_version": "132.0.6834.110",
+      "browser_ttl": 1737992097
+    }
+  ],
   "drivers": [],
   "stats": [
     {
       "browser": "chrome",
       "browser_version": "132",
-      "os": "macos",
+      "os": "linux",
```

Comment:
@alison0 I believe we do not need this. I would try running the tests without this, as it means it will use this for everyone running the tests